docs/tutorial.md

```diff
@@ -690,8 +662,10 @@ int func()
 Let's compile it and see what it looks like.   Here we compile with `-g`
 to include source line info:
 
+> Note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
+
 ```
-> clang -I ../../include -target bpf -Werror -g -O2 -c helpers.c -o helpers.o
+> clang -I .\eBPF-for-Windows.0.4.0\build\native\include\ -target bpf -Werror -g -O2 -c helpers.c -o helpers.o
```

Comment:
This line will need to be updated every time we do another release.  Should add it to the docs\ReleaseProcess.md document and @Alan-Jowett should update #1322 accordingly